artemis-jms-client/src/main/java/org/apache/activemq/artemis/jms/client/compatible1X/ActiveMQTextCompabileMessage.java

```diff
@@ -51,4 +52,10 @@ public ActiveMQTextCompabileMessage(ClientMessage message, ClientSession session
    public ActiveMQTextCompabileMessage(TextMessage foreign, ClientSession session) throws JMSException {
```

Comment:
Not part of this change i know. But spotted spelling mistake. Compabile should be Compatible